internal/wal_show_handler.go

```diff
@@ -0,0 +1,212 @@
+package internal
+
+import (
+	"github.com/wal-g/storages/storage"
+	"github.com/wal-g/tracelog"
+	"github.com/wal-g/wal-g/utility"
+	"sort"
+)
+
+const (
+	TimelineOkStatus          = "OK"
+	TimelineLostSegmentStatus = "LOST_SEGMENTS"
+)
+
+// TimelineInfo contains information about some timeline in storage
+type TimelineInfo struct {
+	Id               uint32          `json:"id"`
+	ParentId         uint32          `json:"parentId"`
```

Comment:
Check that JSON tag case matches with such tags in other parts of wal-g.

Reply:
Changed to snake_case in c97bd4c